### ISSUE TITLE
Calendar popup accessibility

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/webform#257]
For accessibility reasons, the calendar popup was marked as hidden:
- [Add aria-hidden="true" and role="presentation" to calendar icon in date](https://www.drupal.org/project/webform/issues/2534098)

The popup widget doesn't allow keyboard navigation, so it makes sense not to show it for screenreaders. However, it is failing an accessibility audit due to the fact that it's marked hidden but still accepts focus. We should also add `tab-index: -1` to this element. 

[See also this issue, filed in the core queue, that references it](https://github.com/backdrop/backdrop-issues/issues/7048).

**Comment by aitala** [comment]
This fix does resolve the issue that Site Improve has with the calendar popup.

Thanks,
Eric
